.rubocop.yml

```diff
@@ -34,3 +34,6 @@ Rails/LexicallyScopedActionFilter:
 Style/HashSyntax:
   EnforcedShorthandSyntax: never
 
+Lint/UnusedMethodArgument:
+  Exclude:
+    - spec/support/devise_token_auth_overrides.rb  
```

Comment:
Can we fix this lint issue instead of ignore it?

Reply:
Looks like he's taking rails_api_base as a reference

https://github.com/rootstrap/rails_api_base/blob/master/spec/support/devise_token_auth_overrides.rb
https://github.com/rootstrap/rails_api_base/blob/master/.reek.yml#L121
https://github.com/rootstrap/rails_api_base/blob/master/.rubocop.yml#L18